router/src/infer/v3/block_allocator.rs

```diff
@@ -80,9 +124,13 @@ async fn block_allocator_task(
         match cmd {
```

Comment:
Might be unrelated, but why are we using this channel + command pattern when the routine is such a simple function? What's the advantage over just using a `Mutex` or `RwLock`? Current solution seems unnecessarily complicated to me.